nipype/interfaces/bids_utils.py

```diff
@@ -56,21 +48,18 @@ class BIDSDataGrabber(BaseInterface):
     Examples
     --------
 
-    >>> from nipype.interfaces.bids_utils import BIDSDataGrabber
-    >>> from os.path import basename
-
     By default, the BIDSDataGrabber fetches anatomical and functional images
     from a project, and makes BIDS entities (e.g. subject) available for
     filtering outputs.
 
-    >>> bg = BIDSDataGrabber()
+    >>> bg = BIDSDataGrabber() # doctest: +SKIP
```

Comment:
don't skip this.

Reply:
bidsdatagrabber should initialize without pybids, just not run without it.

talking of which it would be useful in run_interface to do:

```
if not have_pybids:
    raise ImportError('Could not import pybids')
```